### PR TITLE
Migrate timeline info to checklist and procedure

### PR DESCRIPTION
**Pull request by pradnyavmw**
# Changes

- Migrated contents from Timeline.md to release-checklist.md and procedure.md
- Deleted Timeline.md as there is lot of duplication

/kind <kind>

Fixes #

**Release Note**

```release-note

```

**Docs**

```docs

```

### CODE REVIEW

**Comment by kvmware** [comment]
looks like you have a conflict else lgtm!

File: PROCEDURES.md [hunk 1]
```diff
@@ -156,19 +178,19 @@ This is because the website references these releases in various locations.
...
 ### Create a release branch
 
 1. Check on the `#docs` Slack channel to make sure the release is ready.
-_In the future, we should automate this so the check isn't needed._
+   _In the future, we should automate this so the check isn't needed._
 
 1. Using the GitHub UI, create a `release-X.Y` branch based on `main`.
 
-  ![branch](https://user-images.githubusercontent.com/35748459/87461583-804c4c80-c5c3-11ea-8105-f9b34988c9af.png)
+![branch](https://user-images.githubusercontent.com/35748459/87461583-804c4c80-c5c3-11ea-8105-f9b34988c9af.png)
```
Comment (dprotaso) [inline PROCEDURES.md:193]: likewise see if we can indent this

**Review by dprotaso — COMMENTED** [review]
You'll want to update this script to modify the checklist instead of the TIMELINE.md

https://github.com/knative/release/blob/main/hack/update-timeline-refs.sh#L1

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 2]
```diff
@@ -4,25 +4,44 @@ about: This checklist tracks tasks and repo releases for a Knative release.
...
+
+## Repos to be released
+
+- All repos to be released are listed in this document
+- Each repo indicates its releasability and other statuses, when applicable
+- Each repo needs to be successfully updated to use the latest version of all dependencies **before** cutting its release branch
 
 ### T-minus 30 days
 
+- Open a [release checklist](https://github.com/knative/release/issues/new?assignees=knative/knative-release-leads&template=release-checklist.yaml) issue.
```
[the remark this answers is not in the document]
Comment (pradnyavmw) [reply .github/ISSUE_TEMPLATE/release-checklist.md]: done

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 3]
```diff
@@ -4,25 +4,44 @@ about: This checklist tracks tasks and repo releases for a Knative release.
...
   - See workflow to check rollout: [![Owner Workflow](https://github.com/knative/release/actions/workflows/check-owners.yaml/badge.svg)](https://github.com/knative/release/actions/workflows/check-owners.yaml)
 
 ### T-minus 14 days
+
 - [ ] Update release version in the following workflows
-	- [ ]  Releasability  (see: [procedure](https://github.com/knative/release/blob/main/PROCEDURES.md#updating-the-releasability-defaults))
-	- [ ] Update Dependency  (see [procedure](https://github.com/knative/release/blob/main/PROCEDURES.md#bump-dependencies-in-auto-update-job))
+
+  - [ ] Releasability (see: [procedure](https://github.com/knative/release/blob/main/PROCEDURES.md#updating-the-releasability-defaults))
+  - [ ] Update Dependency (see [procedure](https://github.com/knative/release/blob/main/PROCEDURES.md#bump-dependencies-in-auto-update-job))
```
[the remark this answers is not in the document]
Comment (pradnyavmw) [reply .github/ISSUE_TEMPLATE/release-checklist.md]: done

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 4]
```diff
@@ -4,25 +4,44 @@ about: This checklist tracks tasks and repo releases for a Knative release.
...
 
 - [ ] An announcement has been made in the [**#knative**](https://app.slack.com/client/T08PSQ7BQ/C04LGHDR9K7) Slack channel that [`knative.dev/pkg`](https://github.com/knative/pkg) will be released in a week
 
-### T-minus 7 days
+### T-minus 7 days - releasing supporting repos
+
+Guidance on [Releasing a repository](../../PROCEDURES.md#releasing-a-repository).
```
[the remark this answers is not in the document]
Comment (pradnyavmw) [reply .github/ISSUE_TEMPLATE/release-checklist.md]: done

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 5]
```diff
@@ -4,25 +4,44 @@ about: This checklist tracks tasks and repo releases for a Knative release.
...
 
 - [ ] Start a draft for the announcement blog post in the [Knative Team Drive](https://drive.google.com/drive/folders/1IbO2UeEl_HsAX0CDEdbWptMKnMAgCdPt)
 - [ ] An announcement has been made in the [**#knative**](https://app.slack.com/client/T08PSQ7BQ/C04LGHDR9K7) Slack channel that the cut of [`knative.dev/pkg`](https://github.com/knative/pkg) is imminent
-- [ ] Check the status of the nightly releases (below) for the various repos and let the [working group leads](https://github.com/knative/community/blob/main/working-groups/WORKING-GROUPS.md) know about failures
+- [ ] Check the status of the nightly releases(the nightly job badge against each core repo in Phase 1 below should give its current status) for the various repos and let the [working group leads](https://github.com/knative/community/blob/main/working-groups/WORKING-GROUPS.md) know about failures and ask them to investigate.Repeat this process throughout the week. This should not be a blocker to start cutting the supporting repositories mentioned below.
+
+Notes
+
+- Some of the supporting repos can be cut parallelly. Refer [Aligning dependencies](../../PROCEDURES.md/#Aligning-dependencies).
```
[the remark this answers is not in the document]
Comment (pradnyavmw) [reply .github/ISSUE_TEMPLATE/release-checklist.md]: done

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 6]
```diff
@@ -4,25 +4,44 @@ about: This checklist tracks tasks and repo releases for a Knative release.
...
 
-### T-minus 7 days
+### T-minus 7 days - releasing supporting repos
+
+Guidance on [Releasing a repository](../../PROCEDURES.md#releasing-a-repository).
 
 - [ ] Start a draft for the announcement blog post in the [Knative Team Drive](https://drive.google.com/drive/folders/1IbO2UeEl_HsAX0CDEdbWptMKnMAgCdPt)
 - [ ] An announcement has been made in the [**#knative**](https://app.slack.com/client/T08PSQ7BQ/C04LGHDR9K7) Slack channel that the cut of [`knative.dev/pkg`](https://github.com/knative/pkg) is imminent
-- [ ] Check the status of the nightly releases (below) for the various repos and let the [working group leads](https://github.com/knative/community/blob/main/working-groups/WORKING-GROUPS.md) know about failures
+- [ ] Check the status of the nightly releases(the nightly job badge against each core repo in Phase 1 below should give its current status) for the various repos and let the [working group leads](https://github.com/knative/community/blob/main/working-groups/WORKING-GROUPS.md) know about failures and ask them to investigate.Repeat this process throughout the week. This should not be a blocker to start cutting the supporting repositories mentioned below.
```
[the remark this answers is not in the document]
Comment (pradnyavmw) [reply .github/ISSUE_TEMPLATE/release-checklist.md]: done

File: PROCEDURES.md [hunk 7]
```diff
@@ -18,90 +31,106 @@ necessary to perform some of them manually.
...
+2. Search for the correct `release-x.y` branch name for the release.
 
-  ![Search for the expected release branch name](images/github-branch-search.png)
+![Search for the expected release branch name](images/github-branch-search.png)
 
-1. Click "Create branch: release-x.y".
+3. Click "Create branch: release-x.y".
 
-  ![Click create branch: release-x.y](images/github-branch-create.png)
+![Click create branch: release-x.y](images/github-branch-create.png)
```
[the remark this answers is not in the document]
Comment (pradnyavmw) [reply PROCEDURES.md]: done

File: PROCEDURES.md [hunk 8]
```diff
@@ -207,21 +231,24 @@ After the client release, the [Homebrew tap](https://github.com/knative/homebrew
...
-* [kn-plugin-source-kafka](https://github.com/knative-sandbox/kn-plugin-source-kafka) is managed via `source-kafka.rb` formula
-* [kn-plugin-source-kamelet](https://github.com/knative-sandbox/kn-plugin-source-kamelet) is managed via `source-kamelet.rb` formula
-* [kn-plugin-quickstart](https://github.com/knative-sandbox/kn-plugin-quickstart/) is managed via `quickstart.rb` formula
-* [kn-plugin-event](https://github.com/knative-sandbox/kn-plugin-event) is managed via `event.rb` formula
+- [kn-plugin-admin](https://github.com/knative-sandbox/kn-plugin-admin) is managed via the `admin.rb` formula
+- [kn-plugin-source-kafka](https://github.com/knative-sandbox/kn-plugin-source-kafka) is managed via `source-kafka.rb` formula
+- [kn-plugin-source-kamelet](https://github.com/knative-sandbox/kn-plugin-source-kamelet) is managed via `source-kamelet.rb` formula
+- [kn-plugin-quickstart](https://github.com/knative-sandbox/kn-plugin-quickstart/) is managed via `quickstart.rb` formula
+- [kn-plugin-event](https://github.com/knative-sandbox/kn-plugin-event) is managed via `event.rb` formula
```
[the remark this answers is not in the document]
Comment (pradnyavmw) [reply PROCEDURES.md]: done

File: PROCEDURES.md [hunk 9]
```diff
@@ -125,20 +155,12 @@ You can manually trigger a release for a repository by re-running its release jo
...
 
-### New release notes document
-For a new Knative release version, start a fresh HackMD release notes document by emptying out the [last release notes document](https://hackmd.io/cJwvzJ4eRVeqqiXzOPtxsA). Post and pin it to the **release-`xdotx`** Slack channel.
-
-### Release notes update
-After a repo release [has been published](#release-job), generate the repo's notes by using the `Release Notes` GitHub Action workflow. See an example in [Eventing](https://github.com/knative/eventing/actions?query=workflow%3A%22Release+Notes%22). The default starting and ending SHAs will work if running out of the `main` branch, or you can determine the correct starting and ending SHAs for the script to run.
+### Exceptions
 
-Use the generated notes to:
+Repos that don't require release artifacts (such as release tags and GitHub release for example) naturally don't need a release note and this step can be skipped for those. All of the [supporting repos](TIMELINE.md#supporting-repos) fit these criteria.
```
[the remark this answers is not in the document]
Comment (pradnyavmw) [reply PROCEDURES.md]: done

File: PROCEDURES.md [hunk 10]
```diff
@@ -125,20 +155,12 @@ You can manually trigger a release for a repository by re-running its release jo
...
 
-1. add them to the [HackMD document](https://hackmd.io/cJwvzJ4eRVeqqiXzOPtxsA)
-2. ask the relevant WG leads if they have any add or edit for the updated repo notes in the [HackMD document](https://hackmd.io/cJwvzJ4eRVeqqiXzOPtxsA)
-3. use the GitHub UI to update the repo release tag with the final notes
+For getting Release Notes for each repo, go to the [workflow](https://github.com/knative/release/actions/workflows/release-note.yaml). Select the appropriate workflow and got to the "Release Notes Summary" section. On the right side, you will see 3 dots. Click on that and you should get a "View raw markdown". Copy the contents of this markdown in the blog-post
```
[the remark this answers is not in the document]
Comment (pradnyavmw) [reply PROCEDURES.md]: done

File: PROCEDURES.md (hunk 1 above)
Comment (pradnyavmw) [reply PROCEDURES.md]: done

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 11]
```diff
@@ -119,148 +156,171 @@ Please review the [TIMELINE](https://github.com/knative/release/blob/main/TIMELI
...
 - [knative-sandbox/kn-plugin-source-kamelet](https://github.com/knative-sandbox/kn-plugin-source-kamelet)
+  - [ ] Nightly Job [![Nightly][kn-plugin-source-kamelet-nightly-badge]][kn-plugin-source-kamelet-nightly-page]
   - [ ] Dependencies up to date - [![releasabilty][kn-plugin-source-kamelet-release-badge]][kn-plugin-source-kamelet-release-workflow]
   - [ ] Branch Cut
   - [ ] Release
-    - Prow Release Job  [![Prow][kn-plugin-source-kamelet-prow-badge]][kn-plugin-source-kamelet-prow-job]
+    - Prow Release Job [![Prow][kn-plugin-source-kamelet-prow-badge]][kn-plugin-source-kamelet-prow-job]
     - Latest Version [![Releases][kn-plugin-source-kamelet-version-badge]][kn-plugin-source-kamelet-release-page]
-  - [ ] Release Notes ([run workflow][release-note-workflow])
+- [ ] Release Notes ([run workflow][release-note-workflow])
```
Comment (dprotaso) [inline .github/ISSUE_TEMPLATE/release-checklist.md:290]: need to fix the indentation of this release note

Comment (pradnyavmw) [reply .github/ISSUE_TEMPLATE/release-checklist.md]: done

**Comment by kvmware** [comment]
/approve
and lgtm. @dprotaso you were last with comments if you happy wanna give it a lgtm?

**Pull request merged**